core/block/simple/dataview/dataview.go

```diff
@@ -516,3 +516,7 @@ func (s *Dataview) UpdateRelationOld(relationKey string, rel model.Relation) err
 
 	return nil
 }
+
+func (d *Dataview) IsEmpty() bool {
```

Comment:
consider adding IsEmpty() to `simple.Block` interface if every block should implement it

also do not forget about simple table block, it's accidentally put in `core/block/editor/table` package, maybe because of some kind of import cycles

Reply:
we may think about it separately. Table is the layout block. It means it in a way always empty itself. We don't consider amount of children for the IsEmpty(). 

For this particular task we need to check if there is any block with it's own non-empty content. If it will be table without any blocks with data inside, we will ignore it